<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MassFontChangeDialog</class>
 <widget class="QDialog" name="MassFontChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change fonts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="LeechCraft::Util::FontChooserWidget" name="FontChooser_" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="StandardBox_">
     <property name="text">
      <string>Standard font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="FixedBox_">
     <property name="text">
      <string>Monospace font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="SerifBox_">
     <property name="text">
      <string>Serif font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="SansSerifBox_">
     <property name="text">
      <string>Sans-serif font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CursiveBox_">
     <property name="text">
      <string>Cursive font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="FantasyBox_">
     <property name="text">
      <string>Fantasy font</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Util::FontChooserWidget</class>
   <extends>QWidget</extends>
   <header>fontchooserwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MassFontChangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MassFontChangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
